src/compiler/compiler.ml

```diff
@@ -321,6 +344,7 @@ let call_light_init_macro com path =
 
 let compile ctx actx callbacks =
 	let com = ctx.com in
+	MacroContext.macro_interp_cache := None;
```

Comment:
I'll have to review this because this still seems strange to me.

Reply:
It was wrong indeed (was wiping `@:persistent` vars between runs for example).
Reworked init macro context creation, now works way better and without that weird trick.